src/current/v24.1/cockroach-cert.md

```diff
@@ -51,12 +51,16 @@ File name pattern | File usage
 `node.crt`   | Server certificate. <br><br>`node.crt` must be signed by `ca.crt` and must have `CN=node` and the list of IP addresses and DNS names listed in `Subject Alternative Name` field. CockroachDB also supports [wildcard notation in DNS names](https://wikipedia.org/wiki/Wildcard_certificate).
 `node.key`   | Key for server certificate.
 
+{{site.data.alerts.callout_info}}
+{% include {{page.version.version}}/misc/cert-auth-using-x509-subject.md %}
```

Comment:
`cockroach cert` doesn't let you control the other subject fields so this include doesn't feel appropriate on this page. (it's also here twice).

Reply:
Makes sense - I was trying to make the other "cert things" link to the subject mapping stuff so people who were e.g. browsing the `cockroach cert` docs and actually needed subject mapping could get bounced over to the subject mapping docs

that said I probably went a little overboard - removed